test/cases/minimal-raw.sh

```diff
@@ -86,10 +86,10 @@ SSH_KEY_PUB=$(cat "${SSH_KEY}".pub)
 EDGE_USER_PASSWORD=foobar
 
 case "${ID}-${VERSION_ID}" in
-    "rhel-8.9")
+    "rhel-8.10")
         OS_VARIANT="rhel8-unknown"
         ;;
-    "rhel-9.3")
+    "rhel-9.4")
```

Comment:
Can this just be a match on `rhel-8*`, `rhel-9*`, etc. If we need specifics for different minor versions we can always add them.